### PR TITLE
Gene view redesign

### PR DESCRIPTION
**Pull request by mdml**
- Rewrite save to use librsvg to convert to PDF/PNG/SVG
- Simplify printing greatly using "data-" attributes in links
- Redesign of the five views, putting them in panels and adding common 
- Use grid layout for views/control panel, and fix control panel in place using Bootstrap's affix
- Add icons to control toggling, saving, printing, instructions, and (eventually) tours

Still to-do: implement tours for "?" icons

### CODE REVIEW

**Comment by johndashen** [comment]
The panels on the main gene view page as a whole look great.  However, I can't see the links in the upper right corner of each panel.  

A few notes:
- In the aberrations view, the legend mouseover is cut off by the panel boundary.
- Also in the aberrations view, the Sort columns[+] text appears twice, once on the left and once on the right.
- The Control Panel > Share link doesn't display a link.

I like how the control panel stays on the same position on the page, which is a nice effect. I wonder if we could also keep the navbar on top of the page while letting only the gene view panels scroll, and do this for other pages in MAGI as well.

**Comment by johndashen** [comment]
Also, this may be from another branch, but on the front page, the example queries (SWI-SNF/TCGA Pancancer, etc.) no longer change the chosen datasets in the dropdown.

File: views/view.jade [hunk 1]
```diff
@@ -5,233 +5,255 @@ block body
...
+          //- CNAs row
+          div(class="row", id="cnasRow")
+            div(class="col-lg-12")
+              div(class="panel panel-default")
+                div(class="panel-heading clearfix")
+                  h4(class="panel-title pull-left")
+                    | Copy Number Aberrations
+                  ul(class="list-inline pull-right")
+                    li
+                      a(class="fa fa-info", title="CNA view description", tabindex="0", data-toggle="popover", data-trigger="hover", data-html="true", data-placement="bottom", data-content="Shows the copy number aberrations surrounding the selected gene (highlighted as red) on the chromosome. The topest panel shows genes (shown as blocks) distributed in the region. You can click and drag a window in this panel to zoom in and see the details in the middle and bottom panels. The middle and bottom panels show genes and copy number aberration inside the zoom in window you just dragged, respectively. Copy number aberrations are colored by cancer type. Each row represents one sample. You can mouseover copy number aberration to get more detail information.")
```
Comment (johndashen) [inline views/view.jade]: topest -> topmost

File: views/view.jade [hunk 2]
```diff
@@ -5,233 +5,255 @@ block body
...
+            
+            //- Transcript view
+            div(class="col-lg-6")
+              div(class="panel panel-default")
+                div(class="panel-heading clearfix")
+                  h4(class="panel-title pull-left")
+                    | Transcript
+                  ul(class="list-inline pull-right")
+                    li
+                      a(class="fa fa-info", title="Transcript view description", tabindex="0", data-toggle="popover", data-trigger="hover", data-html="true", data-placement="bottom", data-content="Shows the mutations in each REFSEQ transcript of each gene in the subnetwork. Mutations are colored by cancer type. Each symbol represents a distinct mutation type. Putatively inactivating mutations are shown below the transcript, and the remaining mutations are shown above the transcript. Select one of the three protein domain databases, including Pfam, SMART, and NCBI Conserve Domain, to show annotated protein domains for each transcript -- scroll over a domain to reveal its name. You can <b>zoom in on and drag the transcript annotation</b> by scrolling and/or clicking the figure with your mouse.")
```
Comment (johndashen) [inline views/view.jade]: The description refers selectors for protein domain databases (Pfam, SMART, etc.).  Are they visible somewhere?

**Comment by mdml** [comment]
Good catches on the typos, I cleared them up.

I also fixed the other three issues you mentioned. I moved sorting into a icon in the aberrations panel. You need to run bower install to install the new jquery-ui dependency, and also need to clone the latest version of gd3.

**Comment by johndashen** [comment]
Looks good now.  I'll merge this into the link-django-annos branch and put
it up on the node MAGI machines.

**Pull request merged**